tests/testthat/test-utilities-data-set.R

```diff
@@ -15,10 +15,10 @@ test_that("It can convert an empty data set", {
   expect_equal(
     dataSetToDataFrame(dataSet),
     data.frame(
-      name = character(0), xValues = logical(0), yValues = logical(0), yErrorValues = logical(0),
+      name = character(0), xValues = numeric(0), yValues = numeric(0), yErrorValues = numeric(0),

```

Comment:
@msevestre This is why care about having the right kind of `NA`! 

Note how `xValues` and `yValues` used to be vectors of length 0 and type `logical` before!

Reply:
the fact that they were logical before and that we tested this and found that ok is what is more of a mystery for me lol
Very cool